app/views/activities/new.html.slim

```diff
@@ -0,0 +1,65 @@
+h1.activity-form-title Add New Activity
+hr
+
+= simple_form_for @activity, as: :activity, url: activities_path,
```

Comment:
Why was this not extracted to a partial? This and `edit.html.slim` should render `_form.html.slim`.